ReactAndroid/src/main/java/com/facebook/react/uimanager/layoutanimation/LayoutAnimationController.java

```diff
@@ -91,4 +100,39 @@ public void applyLayoutUpdate(View view, int x, int y, int width, int height) {
       view.startAnimation(animation);
     }
   }
+
+  /**
+   * Animate a view deletion using the layout animation configuration supplied during initialization.
+   *
+   * @param view     The view to animate.
+   * @param callback Called once the animation is finished, should be used to
+   *                 completely remove the view.
+   */
+  public void deleteView(final View view, final Callback callback) {
```

Comment:
Olivier, should this be part of applyLayoutUpdate? I can't really tell, but I did notice that there's no createView call on this class.